### DISCUSSION TITLE
Ctril + Up/Down on Linux

### DISCUSSION THREAD

**Discussion by andyczerwonka** [vslinko/obsidian-outliner#351] [Q&A]
I'm not getting anything when I try to move lines around in a list on Linux, but notice that the docs only reference Windows and Mac. I'm running Ubuntu 20.04. Has anyone got it to work?

**Comment by vslinko** [comment]
@andyczerwonka I haven't tested the plugin on Linux, but it should work correctly. The plugin uses Obsidian's built-in API for shortcuts, which should work on Linux. Could you check that you don't have any errors in the DevTools Console.

https://github.com/vslinko/obsidian-outliner/blob/main/src/features/MoveItemsFeature.ts#L27-L53

**Reply by andyczerwonka** [reply]
@vslinko I don't know what I did, but it's working. Strange. We can close this ticket.

**Comment by andyczerwonka** [comment]
I don't know what I did, but it's working. Strange.
